keps/sig-node/20181106-in-place-update-of-pod-resources.md

```diff
@@ -172,9 +184,24 @@ available memory are more probable to require restarts.
 If more than one resource type with different policies are updated, then
 RestartContainer policy takes precedence over NoRestart policy.
 
-Additionally, if RestartPolicy is 'Never', ResizePolicy should be set to
+Additionally, if RestartPolicy is 'Never', ResizePolicy must be set to
 NoRestart in order to pass validation.
 
+<<[UNRESOLVED]>>
+Should something like RuntimeClass be allowed to say "no in-place restarts are
+allowed", so that users would get a synchronous error if they set this?  Or is
+that overkill?`
+<<[/UNRESOLVED]>>
+
+<<[UNRESOLVED]>>
+`resizePolicy: NoRestart` is not obvious.  To me that reads as "you may not
```

Comment:
On phrasing:  is there a more self-documenting expression we can use?

Reply:
How about ```LiveUpdate``` or ```LiveResize```?

Reply:
I got a chance to speak with @dchen1107 and @dashpole today.  I'll update more comment threads in a minute.

On this topic we seem to have converged - the semantic is really "I don't need a restart, but a restart might happen anyway".  Cool.

I suck at naming things, so grain-of-salt time.

`resizePolicy: NoRestart` is OK, as long as comments are clear.

`resizePolicy: RestartNotRequired` is better but verbose.  Since it is the default, maybe this is OK?

Reply:
>I suck at naming things, so grain-of-salt time.

That makes two of us :)